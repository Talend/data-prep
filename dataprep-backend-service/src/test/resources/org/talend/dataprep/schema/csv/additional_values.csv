name;email
pierre;pierre@email.com;1970;paris
audrey;audrey@email.com
alistaire;alistaire@email.com;1981
Ulrich;ulrich@email.com;1990;Bonn
