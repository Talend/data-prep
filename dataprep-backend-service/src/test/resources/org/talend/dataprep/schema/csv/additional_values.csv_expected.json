[
  {
    "0000": "pierre",
    "0001": "pierre@email.com;1970;paris"
  },
  {
    "0000": "audrey",
    "0001": "audrey@email.com"
  },
  {
    "0000": "alistaire",
    "0001": "alistaire@email.com;1981"
  },
  {
    "0000": "Ulrich",
    "0001": "ulrich@email.com;1990;Bonn"
  }
]